packages/sw-broadcast-cache-update/src/index.js

```diff
@@ -60,11 +60,11 @@
 import Behavior from './lib/behavior';
 import broadcastUpdate from './lib/broadcast-update';
 import {cacheUpdatedMessageType} from './lib/constants';
-import sameResponses from './lib/responses-are-same';
+import responsesAreSame from './lib/responses-are-same';
 
 export {
   Behavior,
   broadcastUpdate,
   cacheUpdatedMessageType,
-  sameResponses,
+  responsesAreSame,
```

Comment:
I realized when writing the tests that this was being exported with the wrong name...